.travis.yml

```diff
@@ -31,15 +31,15 @@ branches:
   only:
     - master
     - develop
-    - /^release\/.*$/
+    - /^release/
+    - /^travis/
 
-# Additional installs are Valgrind for the memory-tests
-# and gcc-multilib for the compilation of the different
-# architectures.
+# Additional installs: Valgrind for memory tests.
 install:
   - sudo apt-get update -qq
   - sudo apt-get install valgrind
-  - sudo apt-get install gcc-multilib
+  - apt-cache search gcc | grep '^gcc-[0-9\.]* '
+  - apt-cache search clang | grep compiler
```

Comment:
I would install them individually instead. The newer ones won't be a problem but the older ones, like e.g.: gcc-4.7 might. Or set the distribution which would be the safest variant, I think.

Reply:
The compileds are installed separately. The distro is fixed. These lines just print the available compilers in the package database.